tests/test_util.py

```diff
@@ -15,7 +15,7 @@
 
 def cpp_supported():
     """Is cpp (the C preprocessor) supported as a native command?"""
-    return platform.system() == 'Linux'
```

Comment:
This was specifically removed last year due to failures reported on some Macs that don't have gcc/cpp installed by default. It's not clear to me that this should change now

Reply:
On macOS if `cpp` is installed, then `gcc` is installed too. Python on macOS comes with Xcode tools, and is not bundled, so I do not think you can install the former without the latter.